src/components/molecules/NavBar/components/NormalRadio/NormalRadio.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useMemo } from "react";
+import { OverlayTrigger, Popover } from "react-bootstrap";
+import classNames from "classnames";
+
+import { generateId } from "utils/string";
+
+import { RadioModal } from "components/organisms/RadioModal/RadioModal";
+
+import { ButtonNG } from "components/atoms/ButtonNG";
+
+import "./NormalRadio.scss";
+
+interface NormalRadioProps {
+  volume: number;
+  setVolume: (value: ((prevState: number) => number) | number) => void;
+  title?: string;
+  onEnableHandler: () => void;
+  radioPlaying: boolean;
+  defaultShow: boolean;
+}
+
+export const NormalRadio: React.FC<NormalRadioProps> = ({
+  title,
+  defaultShow,
+  onEnableHandler,
+  radioPlaying,
+  volume,
+  setVolume,
+}) => {
+  const id = useMemo(() => generateId("NormalRadio-popover"), []);
+
+  const popoverClasses = classNames({
+    "NormalRadio NormalRadio--on": radioPlaying && volume,
+    "NormalRadio NormalRadio--off": !(radioPlaying && volume),
+  });
+  const buttonClasses = classNames({
+    "NormalRadio__button NormalRadio__button--on": radioPlaying && volume,
+    "NormalRadio__button NormalRadio__button--off": !(radioPlaying && volume),
+  });
```

Comment:
```suggestion
  const buttonClasses = classNames({
    "NormalRadio__button NormalRadio__button--on": radioPlaying && volume,
    "NormalRadio__button NormalRadio__button--off": !(radioPlaying && volume),
  });
```
Same here

Reply:
For this change, please first provide a reference to https://github.com/sparkletown/sparkle/blob/staging/docs/Questions.md (as per this precedent https://github.com/sparkletown/sparkle/pull/2219#discussion_r713837478 )